ods_ci/tasks/Resources/Provisioning/Hive/provision.robot

```diff
@@ -145,31 +193,43 @@ Watch Hive Install Log
     Should Contain    ${new_log_data}    install completed successfully
 
 Wait For Cluster To Be Ready
-    ${pool_namespace} =    Get Cluster Pool Namespace    ${pool_name}
-    Set Task Variable    ${pool_namespace}
-    Log    Watching Hive Pool namespace: ${pool_namespace}    console=True
+    IF    ${use_cluster_pool}
+        ${pool_namespace} =    Get Cluster Pool Namespace    ${pool_name}
+        Set Task Variable    ${pool_namespace}
+        Set Task Variable    ${clusterdeployment_name}    ${pool_namespace}
```

Comment:
@bdattoma FYI:
- The Cluster deployment name has a similar name (if I recall) as the cluster pool namespace by default.
- The Cluster pool namespace is a unique name, which includes the cluster name prefix.
- We should not make the cluster name equal to cluster pool namespace (which is currently the case for GCP and AWS).

Reply:
I just replicated the logic that was there earlier.
The code was using `${pool_namespace}` variable as cluster deployment name when using cluster pools.